app/models/challenge.rb

```diff
@@ -182,4 +183,8 @@ class Challenge < ApplicationRecord
   scope :open, -> { where(sub_status: 'open') }
   scope :opening_soon, -> { where(status: 'published', sub_status: nil) }
   scope :closed, -> { where(sub_status: 'closed') }
+
+  def archived?
+    status == "archived"
```

Comment:
this is `sub_status` on phoenix
```suggestion
    sub_status == "archived"
```